src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.cs

```diff
@@ -425,7 +425,7 @@ private TryUpdateSourceResults TryUpdateSource()
 
             // check to see if name has already been added
             // also make sure it's not the same as the aggregate source ('All')
-            bool hasName = sourcesList.Any(ps => ps != selectedPackageSource && string.Equals(name, ps.Name, StringComparison.CurrentCultureIgnoreCase));
```

Comment:
I decided not to make this change in this file, revert the changes. It brings some unwanted side effects, if update existing source `encyclopaedia` with `Encyclopædia` on PMUI then save it creates duplicate entrees in nuget.config file.
Problem is when saving it check existing using packagesource key but due existing following code it assumes above 2 are different hence creates another entry instead of updating it. https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Configuration/Settings/Items/SourceItem.cs#L80